java/ArrayReverse.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Arrays;
+
+public class ArrayReverse {
+    public static void main(String[] args) {
+
+        int[] arrayOne = {1,2,3,4,5,6};
```

Comment:
This is great, you created three different arrays to run the same method on. You haven't been taught how to test with IntelliJ, but it shows you are a test driven dev!